HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -820,6 +820,20 @@ def self.apply_appliances(hpxml, nbeds, eri_version)
       if clothes_dryer.usage_multiplier.nil?
         clothes_dryer.usage_multiplier = 1.0
       end
+      if clothes_dryer.is_vented.nil?
+        clothes_dryer.is_vented = true
+      end
+      if clothes_dryer.is_vented && clothes_dryer.vented_flow_rate.nil?
+        if not clothes_dryer.combined_energy_factor.nil?
+          if clothes_dryer.combined_energy_factor < 4.5
+            clothes_dryer.vented_flow_rate = 100.0
+          end
+        elsif not clothes_dryer.energy_factor.nil?
+          if clothes_dryer.energy_factor < HotWaterAndAppliances.calc_clothes_dryer_ef_from_cef(4.5)
+            clothes_dryer.vented_flow_rate = 100.0
+          end
+        end
```

Comment:
I previously left a comment on this, but you [marked it as resolved without addressing it](https://github.com/NREL/OpenStudio-HPXML/pull/464#discussion_r469366111). 

The code makes no sense for two reasons:
1. If there is a dryer specified as vented but without a flow rate, and its CEF > 4.5, you end up with a nil flow rate. That's not appropriate and probably results in an error in airflow.rb. If you really wanted to do this, you should set `is_vented` based on CEF.
2. But I don't think any of the defaults _should_ be set based on CEF. Heat pump clothes dryers can be vented or unvented. I suggest assuming 100 cfm whenever a dryer is vented, period.